.travis.yml

```diff
@@ -14,6 +16,7 @@ matrix:
           script:
               - rustup component add rustfmt
               - rustup component add rust-src
+              - printf "\n\n[profile.dev]\ndebug = false\nopt-level = 1\n" >> Cargo.toml
```

Comment:
:cry:

Reply:
Putting bash into YAML is definitely not my favorite thing :D

Reply:
I'm still running tests for #1561 :laughing:.